src/material-experimental/mdc-autocomplete/animations.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  animate,
+  AnimationTriggerMetadata,
+  group,
+  state,
+  style,
+  transition,
+  trigger
+} from '@angular/animations';
+
+// Animation values come from
+// https://github.com/material-components/material-components-web/blob/master/packages/mdc-menu-surface/_mixins.scss
```

Comment:
Some entry points like `@material/ripple` export a `numbers` consta. Maybe the menu surface has something similar?

Reply:
Looks like I can get some of them that way, one of the numbers comes from the @material/animations package which doesn't seem to have an equivalent. Also can't get the timing curves. In the future when we drop IE11 we can read the CSS vars

Reply:
For some reason using the constants doesn't work in SSR. I'm just going to leave them hardcoded for now, with the comment.